src/Engine/Network/Server.cpp

```diff
@@ -669,3 +671,12 @@ PlayerID Server::getPlayerIDFromEntityID(EntityID entityID)
     }
     return -1;
 }
+
+void Server::popNetworkSegmentOfHeader(Packet & packet)
```

Comment:
This and Client::popNetworkSegmentOfHeader can probably be moved up to Network class.